lib/OpenQA/Task/Job/FinalizeResults.pm

```diff
@@ -42,25 +42,38 @@ sub _finalize_results {
 }
 
 sub _run_hook_script ($minion_job, $openqa_job, $app, $guard) {
-    my $trigger_hook = $openqa_job->settings_hash->{_TRIGGER_JOB_DONE_HOOK};
+    my $settings = $openqa_job->settings_hash;
+    my $trigger_hook = $settings->{_TRIGGER_JOB_DONE_HOOK};
+
     return undef if defined $trigger_hook && !$trigger_hook;
     return undef unless my $result = $openqa_job->result;
+
     my $hooks = $app->config->{hooks};
     my $key = "job_done_hook_$result";
     my $hook = $ENV{'OPENQA_' . uc $key} // $hooks->{lc $key};
     $hook = $hooks->{job_done_hook} if !$hook && ($trigger_hook || $hooks->{"job_done_hook_enable_$result"});
     return undef unless $hook;
+
     my $timeout = $ENV{OPENQA_JOB_DONE_HOOK_TIMEOUT} // '5m';
     my $kill_timeout = $ENV{OPENQA_JOB_DONE_HOOK_KILL_TIMEOUT} // '30s';
+    my $delay = $settings->{_TRIGGER_JOB_DONE_DELAY} // $ENV{OPENQA_JOB_DONE_HOOK_DELAY} // 60;
```

Comment:
```suggestion
    my $delay = $settings->{_TRIGGER_JOB_DONE_DELAY} // $ENV{OPENQA_JOB_DONE_HOOK_DELAY} // '1m';
```

right?

Reply:
No, it's seconds, that's what the lower level `delay` option uses. Are we already using a module to convert `1m` to `60` somewhere else?

Reply:
right, sorry. It must be

```suggestion
    my $delay = $settings->{_TRIGGER_JOB_DONE_DELAY} // $ENV{OPENQA_JOB_DONE_HOOK_DELAY} // ONE_MINUTE;
```

using the Time::Seconds module, see https://github.com/os-autoinst/openQA/pull/4718/files#diff-10b9cc653a113467d85dbb952ef075b65f314369090682d3f1094654e2be0b2aR7

Reply:
Ah, sure.